src/applications/gi/components/vet-tec/VetTecProgramSearchResult.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { Link } from 'react-router';
+
+import { formatCurrency, locationInfo } from '../../utils/helpers';
+import { renderPreferredProviderFlag } from '../../utils/render';
+
+class VetTecProgramSearchResult extends React.Component {
+  render() {
+    const { version, result } = this.props;
+    const {
+      facilityCode,
+      description,
+      institutionName,
+      city,
+      state,
+      country,
+      tuitionAmount,
+      lengthInWeeks,
+      vaBah,
+      dodBah,
+    } = result;
+
+    const housing = Math.min(dodBah, vaBah);
+
+    const linkTo = {
+      pathname: `profile/${facilityCode}/${description}`,
+      query: version ? { version } : {},
+    };
+
+    return (
+      <div className="search-result">
+        <div className="outer">
+          <div className="inner">
+            <div className="row">
+              <div className="small-12 usa-width-seven-twelfths medium-7 columns">
```

Comment:
I would suggest using the [layout classes from the design system](https://design.va.gov/layout/flexbox-grid) whenever possible, but I understand if the surrounding markup is still using the Foundation classes.